lib/dotloop/errors/client/forbidden.rb

```diff
@@ -0,0 +1,8 @@
+module Dotloop
```

Comment:
This is the canonical and correct way.
However, can we define the error classes at the Gem level as in:
https://github.com/jnunemaker/httparty/blob/master/lib/httparty/exceptions.rb

This seems to be accepted practice in gems.